test/lib/sql.js

```diff
@@ -24,12 +25,10 @@ describe(`SQLite worker wrapper`, () => {
 		await database.run(insert, ['a', 'b']);
 	});
 	it(`should support both inline and object params`, async () => {
-		const num = await database.prepare(`INSERT INTO test (col, col2) VALUES($col, $col2)`);
 		await database.run(insert, ['a', 'b']);
-		await database.run(num, {col: 'a', col2: 'b'});
+		await database.run(`INSERT INTO test (col, col2) VALUES($col, $col2)`, {col: 'a', col2: 'b'});
 	});
 	it(`should retrieve one line from Database.get`, async () => {
-		const result = await database.get(select);
-		assert(!!result, 1);
+		assert(!!await database.get(select), 1);
```

Comment:
What does `1` mean here? The second arg to `assert` is a description of what the bool means, isn't it?